### PR TITLE
PLT-1796 Refactor and modularize analytics on the client

### PR DESCRIPTION
**Pull request by jwilander**

### CODE REVIEW

File: store/sql_session_store.go [hunk 1]
```diff
@@ -255,3 +255,32 @@ func (me SqlSessionStore) UpdateDeviceId(id, deviceId string) StoreChannel {
...
+		result := StoreResult{}
+
+		query :=
+			`SELECT
+                COUNT(*)
+            FROM
+                Sessions`
+
+		if len(teamId) > 0 {
+			query += " WHERE TeamId = :TeamId"
```
Comment (coreyhulen) [inline store/sql_session_store.go]: This will include expired sessions?  if a user never logs back in it won't get deleted from the db.

Comment (jwilander) [inline store/sql_session_store.go]: Ah, I missed that. Updating the query..

**Comment by jwilander** [comment]
@coreyhulen ready for re-review. I also added the changes to the portuguese json

File: web/react/components/analytics/system_analytics.jsx [hunk 2]
```diff
@@ -0,0 +1,346 @@
...
+
+        return false;
+    }
+
+    onChange() {
+        this.setState({stats: AnalyticsStore.getAllSystem()});
+    }
+
+    render() {
+        const stats = this.state.stats;
```
Comment (crspeller) [inline web/react/components/analytics/system_analytics.jsx]: Will this render if stats are not available?

Comment (jwilander) [inline web/react/components/analytics/system_analytics.jsx]: Each stat component can handle null as a value so it should work fine

**Comment by crspeller** [comment]
+1

**Pull request merged**